### PR TITLE
test: Add CI scenarios for eBPF Chart. #964

### PR DESCRIPTION
**Pull request by hyfj44255**
# Description
Currently the ebpf chart has no CI tests. we should add a test for the default values.yaml at least, but it would be best if we could have more comprehensive coverage.

Fixes #(issue)
https://github.com/open-telemetry/opentelemetry-helm-charts/issues/964

## Type of change
- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Go to Contrib repo root directory.
```
cd opentelemetry-helm-charts
```

Create a kind cluster same as the one in repo's Github action
```
kind create cluster --name ebpf-cluster --config ./.github/kind-1.24.yaml

```
Run CT to preform the test
```
ct install --charts charts/opentelemetry-ebpf
```

### CODE REVIEW

File: charts/opentelemetry-ebpf/ci/enabled-log-debug-values.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+# enable --log-console
+log:
+  console: true
+  level: trace
+
+debug:
+  enabled: true
+  storeMinidump: true
+  sendUnplannedExitMetric: true
```
Comment (TylerHelmuth) [inline charts/opentelemetry-ebpf/ci/enabled-log-debug-values.yaml:13]:
```suggestion
  sendUnplannedExitMetric: true

```

File: charts/opentelemetry-ebpf/ci/requests-limits-values.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        memory: "256Mi"
+        cpu: "300m"
+  watcher:
+    resources:
+      requests:
+        memory: "128Mi"
+        cpu: "200m"
+      limits:
+        memory: "256Mi"
+        cpu: "300m"
```
Comment (TylerHelmuth) [inline charts/opentelemetry-ebpf/ci/requests-limits-values.yaml:63]:
```suggestion
        cpu: "300m"

```

File: charts/opentelemetry-ebpf/templates/k8s-collector-deployment.yaml [hunk 3]
```diff
@@ -119,7 +119,7 @@ spec:
           - name: "EBPF_NET_CRASH_METRIC_HOST"
             value: {{ .Values.endpoint.address }}
           - name: "EBPF_NET_CRASH_METRIC_PORT"
-            value: {{ .Values.endpoint.port }}
```
Comment (nicolastakashi) [inline charts/opentelemetry-ebpf/templates/k8s-collector-deployment.yaml:122]: Let's take this changes to another PR wdyt?

Comment (hyfj44255) [reply charts/opentelemetry-ebpf/templates/k8s-collector-deployment.yaml:122]: Thanks for review,I'd agree let me move this out, one pr should do one thing

**Comment by hyfj44255** [comment]
Hi @nicolastakashi I've updated the pr according to you suggestion would it be ok if you take a look again?

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**